nightfall-deployer/truffle-config.js

```diff
@@ -98,7 +98,7 @@ module.exports = {
       timeoutBlocks: 2000,
       skipDryRun: true,
       websockets: true,
-      gasPrice: config.WEB3_OPTIONS.gasPrice,
+      gasPrice: 20000000000,
```

Comment:
Is there a reason why we made this change?

Reply:
Ah, sorry. I pushed local changes in that case. If we use slither, as is compiling through truffle he complains about the env variable. So to run properly slither I had to put a constant while doing the analysis.